Contest year deduction improvement

KaProDes opened the pull request:
*Fixed a minor bug that caused unforced exits.
*Contest Start Year will now be automatically deduced from the datetime.datetime.now() like before but with a minor change based on the month [Removed Year Hardcoding]

KaProDes committed:
Minor bug fixes

KaProDes committed:
Removed Hard-coded Year

*The year of the contest can now be figured from the current datetime. 
(If month is 9 to 12, start of contest year is in current_year, if month is 1 or 2, the start of contest year is current_year-1)

KaProDes committed:
pip install -r requirements.txt

Installs all necessary dependencies.

SnowyKay commented:
Added requirement.txt which one can directly run pip install -r requirements.txt over to install dependencies.

0x48piraj reviewed (changes requested):
`str(datetime.datetime.now())` is not recommended way of parsing `datetime` format, use in-built functions.

KaProDes committed:
Use of built-in function to improve performance.

Replaced str(datetime.datetime.now()).split()[index] with now.month and now.year.
--Better practice--

SnowyKay commented:
@0x48piraj It's been handled. Replaced with now.moth and now.year.
Please merge.
Threading will be added VERY soon.
